srcpkgs/clion/template

```diff
@@ -0,0 +1,58 @@
+# Template file for 'clion'
+_dlname="CLion"
+_pkgname="clion"
+pkgname=clion
+version=2019.1.2
+revision=1
+archs="x86_64"
+depends="virtual?java-environment giflib libXtst"
+short_desc="CLion: A smart cross-platform IDE for C and C++"
+maintainer="Anton Afanasyev <anton@doubleasoftware.com>"
+license="Commercial"
+homepage="https://www.jetbrains.org/"
+distfiles="https://download.jetbrains.com/cpp/${_dlname}-${version}.tar.gz"
+checksum=e8efbf3ec4aed7ef1e0eb8139e799fb2975e4c6363f2ac7504864f96d5aa8e74
+repository=nonfree
+nopie=yes
+
+post_extract() {
+	rm -rf _tmp
+	mkdir _tmp
+	# JetBrains' JRE can be installed separately as a subpackage.
+	# For now, just move it out of the way
+	mv jre64 _tmp/
```

Comment:
why not just remove/not install that?

Reply:
I intended to make further changes to this, to install the jre separately.
But I'll want to create a separate package instead, so this isn't needed, and I removed it -- thanks.